picard/ui/ui_options_renaming.py

```diff
@@ -163,7 +163,7 @@ def retranslateUi(self, RenamingOptionsPage):
         self.move_files.setTitle(_("Move files when saving"))
         self.label.setText(_("Destination directory:"))
         self.move_files_to_browse.setText(_("Browse..."))
-        self.move_additional_files.setText(_("Move additional files:"))
+        self.move_additional_files.setText(_("Move additional files(case insensitive):"))
```

Comment:
Please add a space before the first parenthesis: `"Move additional files (case insensitive):"`